src/odbc_connection.cpp

```diff
@@ -2631,6 +2659,19 @@ bind_buffers
     // bind depending on the column
     switch(column->DataType) {
 
+      // LONG data types should be retrieved through SQLGetData and not
+      // SQLBindCol/SQLFetch, as the buffers for SQLBindCol could be absurd
+      // sizes for small amounts of data transferred.
+      case SQL_WLONGVARCHAR:
+      case SQL_LONGVARCHAR:
+      case SQL_LONGVARBINARY:
+      {
+        column->is_long_data = true;
+         column->bind_type = 
+          (column->DataType == SQL_WLONGVARCHAR ? SQL_C_WCHAR : SQL_C_CHAR);
```

Comment:
So we bind binary data as SQL_C_CHAR not SQL_C_BINARY?

Reply:
Nope, that can't be right.